util/generate-network-configurations.py

```diff
@@ -0,0 +1,161 @@
+from troposphere import Equals, GetAtt, If, Not, Output, Parameter, Ref, Template
+from troposphere.ec2 import (
+    EIP,
+    InternetGateway,
+    NatGateway,
+    Route,
+    RouteTable,
+    Subnet,
+    SubnetRouteTableAssociation,
+    VPCGatewayAttachment,
+)
+
+
+class NetworkHandler:
```

Comment:
also here let's pick  a better class name and filename. Also let's start better organising files under util dir.